components/PostCard.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Post } from '../interfaces/post';
+
+interface IPostProps {
+    post: Post;
+}
+
+export function PostCard({ post }: IPostProps) {
+    function dateFormatter(postDate: Date): string {
+        const date = new Date(postDate);
+        const dd = date.getDate();
+        const mm = date.getMonth() + 1;
+        const yyyy = date.getFullYear();
+
+        return `${mm < 10 ? '0' + mm : mm}/${dd < 10 ? '0' + dd : dd}/${yyyy}`;
```

Comment:
seperate the fields into its own values
```
const displayMonth = `${mm < 10 && '0'}${mm}`;
const displayDay = `${dd < 10 && '0'}${dd}`

return `${displayMonth}/${displayDate}/${yyyy}`;
```

As you can see this is much readable than lumping all of the concats and logic in one line